xml/System.IO/StreamReader.xml

```diff
@@ -1513,10 +1513,12 @@ Following a call to <xref:System.IO.StreamReader.Close%2A>, any operations on th
         <Parameter Name="buffer" Type="System.Span&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">When this method returns, contains the specified span of characters replaced by the characters read from the current source.</param>
+        <summary>Reads the characters from the current stream into a span.</summary>
+        <returns>The number of characters that have been read, or 0 if at the end of the stream and no data was read. The number will be less than or equal to the <paramref name="buffer" /> length, depending on whether the data is available within the stream.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.IO.IOException">The number of characters read from the stream is larger than the <paramref name="buffer" /> length.</exception>
+        <exception cref="T:System.ArgumentNullException"><paramref name="buffer" /> is <see langword="null" />.</exception>
```

Comment:
I'd move the ones that start with a tag to a new line just to avoid having the CI touching this without a "real" update. We should perhaps file a customer suggestion to @joelmartinez to improve this.

```suggestion
        <exception cref="T:System.ArgumentNullException">
          <paramref name="buffer" /> is <see langword="null" />.</exception>
```